capstone/src/main/webapp/dashboard.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Dashboard</title>
+    <link rel="stylesheet" href="/style.css">
+    <!-- TODO(biancamacias): revist url to add api key -->
+    <!-- <script src="https://maps.googleapis.com/maps/api/js?key="></script> -->
+    <script src="/script.js"></script>
```

Comment:
same comment here as with the style sheet one :)

Reply:
i was going to pull some more PRs adding those files.

Reply:
If you want to do that, then rebase this PR off of that branch...